### PR TITLE
Added virt-install kernel and initrd params support

### PR DESCRIPTION
**Pull request by sathnaga**
Added support for using custom kernel, initrd and
kernel args params to virt-install command line

This will help us booting VM with custom
complied kernel

### CODE REVIEW

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -978,6 +1004,13 @@ def add_memballoon(help_text, memballoon_model):
             else:
                 virt_install_cmd += " --boot emulator=%s" % emulator_path
 
+        kernel = params.get("kernel", None)
+        initrd = params.get("initrd", None)
+        kernel_args = params.get("kernel_args", None)
+        if (kernel or initrd) and kernel_args:
+            virt_install_cmd += add_kernel(help_text, virt_install_cmd, kernel,
```
Comment (balamuruhans) [inline virttest/libvirt_vm.py]: good one.

File: virttest/libvirt_vm.py [hunk 2]
```diff
@@ -659,6 +659,32 @@ def add_memballoon(help_text, memballoon_model):
             logging.debug("vm.add_memballoon returning: %s", result)
             return result
 
+        def add_kernel(help_text, cmdline, kernel_path=None, initrd_path=None,
+                       kernel_args=None):
+            """
+            Adding Custom kernel option to boot.
+
+            : param help_text: string, virt-install help text
+            : param kernel_path: string, custom kernel path.
```
Comment (balamuruhans) [inline virttest/libvirt_vm.py]: doc string for cmdline can be added, also `: param kernel_path:` can be `:param kernel_path:` for all

Comment (sathnaga) [inline virttest/libvirt_vm.py]: Sure :-)

**Comment by sathnaga** [comment]
@balamuruhans Fixed :+1: Tnx.

**Comment by sathnaga** [comment]
@will-Do @balamuruhans Thanks :-).
@will-Do  please merge if no further comments

**Comment by sathnaga** [comment]
@xutian @will-Do pls merge if no further comments.

**Comment by sathnaga** [comment]
Thanks @will-Do :+1:

**Pull request merged**